centreon/www/front_src/src/Extensions/Listing/index.tsx

```diff
@@ -374,6 +374,7 @@ const ExtensionsManager = ({ reloadNavigation }: Props): JSX.Element => {
         <Stack direction="row" spacing={2}>
           <Button
             color="primary"
+            data-cy="update-all"
```

Comment:
why cypress specific data attr?  could the generic `data-testid` do the same trick ?

Reply:
@g10 it just seemed logic to me to use this then the oher one

Reply:
@g10 its done